geopandas/tests/test_geodataframe.py

```diff
@@ -490,7 +490,6 @@ def test_from_postgis_custom_geom_col(self):
         validate_boro_df(df, case_sensitive=False)
 
     @pytest.mark.cython
```

Comment:
you can remove the mark as well if you remove the xfail (I only added it be able to run all xfails that are related to the cython branch (there are some other xfails as well))